pom.xml

```diff
@@ -84,11 +87,21 @@
       <artifactId>log4j</artifactId>
       <version>${log4j.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-common</artifactId>
```

Comment:
Why do you need to depend on `hadoop-common`?

Reply:
Without it I get `java.lang.ClassNotFoundException: org.apache.hadoop.conf.Configuration`. In general, I believe Hive JDBC requires some Hadoop dependencies, as discussed in eg https://issues.apache.org/jira/browse/HIVE-15110. Not sure if it's only configuration. Do you have a cleaner solution?

Reply:
Ah, thanks for the info.